vsintegration/tests/UnitTests/InlineParameterNameHintTests.fs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace VisualFSharp.UnitTests.Editor.Hints
+
+open NUnit.Framework
+open HintTestFramework
+
+module InlineParameterNameHintTests =
```

Comment:
How does it work with functions defined using tupled-syntax, as opposed to curried?

E.g. this:

```
    let code = """
let greet (friend1 ,friend2) = $"hello {friend1} and {friend2}"
let greeting = greet ("Liam" ,"Noel")
"""

```

Reply:
Seems to work naturally in a similar manner. Added a test